### ISSUE TITLE
FP 'selector-type-no-unknown' for 'foreignObject' from svg

### ISSUE THREAD

**Issue by vilchik-elena** [stylelint/stylelint#4472]
There are false positives for 'foreignObject' selector for rule 'selector-type-no-unknown'.
```css
.test foreignObject {
  color: white
}
```

With config:
```json
{
  "rules": {
    "selector-type-no-unknown": true
  }
}
```

To reproduce I used https://stylelint.io/demo, following issue is reported
```
1:7 error Unexpected unknown type selector "foreignObject" (selector-type-no-unknown)
```

There should be no issue as [`foreignObject` tag is part of SVG definition](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject).
